cms/templates/home_page.html

```diff
@@ -17,13 +17,20 @@
   {{ block.super }}
   <script type="text/javascript" src="//cdn.jsdelivr.net/npm/slick-carousel@1.8.1/slick/slick.min.js"></script>
   <script type="text/javascript" src="https://cdn.jsdelivr.net/npm/hls.js@latest"></script>
+  <script src="{% static 'js/jquery.fancybox.min.js' %}"></script>
   <script src="{% static 'js/hero.js' %}"></script>
   <script src="{% static 'js/testimonials_carousel.js' %}"></script>
   <script src="{% static 'js/courseware_carousel.js' %}"></script>
   <script src="{% static 'js/text_video_section.js' %}"></script>
   <script src="{% static 'js/image_carousel.js' %}"></script>
 {% endblock %}
 
+{% block extrahead %}
+  {{ block.super }}
+  {# This is included separately because FancyBox isn't used anywhere else so putting into the styles bundle isn't optimal. #}
+  <link  href="{% static 'css/jquery.fancybox.min.css'  %}" rel="stylesheet">
```

Comment:
@ahmed-arbisoft I was thinking to include them via `CDN` instead of downloading. thoughts ?

Reply:
Fine by me :+1:  go for it